sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/AzureMonitorExporterOptions.cs

```diff
@@ -51,5 +51,7 @@ public enum ServiceVersion
         public string StorageDirectory { get; set; }
 
         public bool DisableOfflineStorage { get; set; }
+
+        internal int StatsbeatInterval { get; set; }
```

Comment:
Consider making this a `TimeSpan`. That would eliminate confusion about the unit here (milliseconds, minutes, etc).